runtime/nvqir/NVQIR.cpp

```diff
@@ -175,9 +175,23 @@ void __quantum__rt__resetExecutionContext() {
 
 /// @brief QIR function for allocated a qubit array
 Array *__quantum__rt__qubit_allocate_array(uint64_t size) {
+  std::vector<std::complex<double>> initState(std::pow(2, size), 0.0 + 0.0j);
```

Comment:
This is the starting point for the state vector in this proof of concept. In reality, this NVQIR function would likely get overloaded and we would be grabbing the state data through the bridge (probably as a pointer).